test/interaction_events.test.js

```diff
@@ -84,7 +84,8 @@ function runTests() {
       });
 
       firedWith(t, 'draw.selectionchange', {
-        features: [pointA]
+        features: [pointA],
+        points: []
```

Comment:
@ChrisBellew it would be good to have a test that confirms that points are being emitted. We can file this as a follow up task if you want. The code for interaction tests is a bit wonky.